### PR TITLE
Fakehd

### PR DESCRIPTION
**Pull request by benlumley**
Take2.... 

I have rebased it onto your config branch, and switched enable/disable to config, and then re-squashed it. 

Individual commits are on this branch if you need to see them:

https://github.com/benlumley/msp-osd/tree/spread-osd-devel

Not tested the enable via config other than that it compiles; not got stuff with me right now - but hopefully the rebase saves you a bit of work....

### CODE REVIEW

File: jni/osd_dji_overlay_udp.c [hunk 1]
```diff
@@ -126,6 +139,92 @@ static display_info_t *current_display_info;
...
 
+#define FAKEHD_ENABLE_KEY "fakehd_enable"
+static int fakehd_enabled = 0;
+static int fakehd_trigger_x = 99;
+static int fakehd_trigger_y = 99;
+
+static void check_is_fakehd_enabled()
+{
+    DEBUG_PRINT("checking for fakehd\n");
+    if (get_boolean_config_value(SPLASH_KEY))
```
Comment (bri3d) [inline jni/osd_dji_overlay_udp.c]: 🤔 - I'll fix this when I rebase over the merged config branch

Comment (benlumley) [reply jni/osd_dji_overlay_udp.c]: Dammit. 

Took me about 10 minutes looking to spot what I'd done!

**Comment by benlumley** [comment]
Rebased on your newer json config branch / pushed / resquashed

**Comment by benlumley** [comment]
Branch with individual commits that this was squashed onto this branch is here still:

https://github.com/benlumley/msp-osd/tree/spread-osd-devel

And if you prefer to rebase yourself, branch prior to rebase on json-config (and minus the final commit that converts to json-config to enable) is here:

https://github.com/benlumley/msp-osd/tree/spread-osd-on-main

**Pull request closed without merging**